<ui version="4.0" >
 <class>IngredientDialog</class>
 <widget class="QDialog" name="IngredientDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ingredient</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <layout class="QFormLayout" name="formLayout" >
       <property name="fieldGrowthPolicy" >
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="buddy" >
          <cstring>ingredientName</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QCheckBox" name="ingredientHave" >
         <property name="text" >
          <string>Have</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Description:</string>
         </property>
         <property name="buddy" >
          <cstring>ingredientDescription</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QTextEdit" name="ingredientDescription" />
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="ingredientName" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QPushButton" name="ingredientImage" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="resources.qrc" >
           <normaloff>:/main/pixmaps/noimage.png</normaloff>:/main/pixmaps/noimage.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="noIngredientImage" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Unset Image</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ingredientName</tabstop>
  <tabstop>ingredientHave</tabstop>
  <tabstop>ingredientDescription</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>ingredientImage</tabstop>
  <tabstop>noIngredientImage</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IngredientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel" >
     <x>373</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IngredientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel" >
     <x>373</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ingredientName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>IngredientDialog</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel" >
     <x>211</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ingredientImage</sender>
   <signal>clicked()</signal>
   <receiver>IngredientDialog</receiver>
   <slot>findImage()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>379</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel" >
     <x>431</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noIngredientImage</sender>
   <signal>clicked()</signal>
   <receiver>IngredientDialog</receiver>
   <slot>noImage()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel" >
     <x>333</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findImage()</slot>
  <slot>noImage()</slot>
 </slots>
</ui>
